webui/react/src/e2e/models/components/F_ExperiementList.ts

```diff
@@ -37,4 +32,8 @@ class ExperimentRow extends Row<ExperimentRow, ExperimentHeadRow> {
     super(args);
     this.columnPositions.set('ID', 50);
   }
+  readonly experimentActionDropdown = new ExperimentActionDropdown({
+    openMethod: this.rightClick.bind(this),
```

Comment:
Using [this ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind)as reference, why are we using `bind` here? Isn't this equivalent to the simpler  `openMethod: this.rightClick`

Reply:
Oh is this supposed to have the second `this` be the ExperimetnActionDropdown? I've been trying to figure out where on the base classes `rightClick` was until I saw it on the experiment list. I think at least a comment explaining how the closure here is supposed to work would be helpful.

Reply:
i'll leave a comment. copilot actually suggested this to me, and i left it out. but when i ran the tests, it turns out i needed it. without the `bind` call, we would've performed `ExperimentActionDropdown.rightClick`, which would exist and fail.

Reply:
it's kind of equivalent to
```
openMethod: async () => this.rightClick(),
```